app/build.gradle

```diff
@@ -30,6 +30,13 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    sourceSets {
+        main {
+            assets {
+                srcDirs 'src/main/assets'
```

Comment:
@DiegoA-coder why this code is required? I want to know why I don't understand this kotlin world 😢

Reply:
I insert it automatically jajaja